### PR TITLE
Proposed docs update for OT 

### PR DESCRIPTION
**Pull request by gcg-foundries**
To do
- Diagram
- Dongle stuff

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 683d7f83ea274fee19264c30bef5852003f02dc5 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/288/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 196f28c33a704a066571d6331339461c83fb30e1 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/289/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for a5e1cfa603b261ef8525c1d2a0e1838607ed3204 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/290/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for c352f95883d0621c629101e0c048cf27d4d9cea0 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/291/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for f0db3f5f89304a7b7d528b74b4c6501120f773d3 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/292/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 64891be858d5354d33d04e4ac87c437d8f508bba are browesable at: https://ci.foundries.io/projects/fio-docs/builds/293/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 13d64c29a9f6e84eaa679a8ebf3124d1475ddfdd are browesable at: https://ci.foundries.io/projects/fio-docs/builds/294/docs/artifacts/html/index.html

**Comment by gcg-foundries** [comment]
Andy ... I've uploaded the new system diagram image (same file name .png extension rather than .svg) but the site rendering still shows the old one (and I've cleared my caches). Do you need to do any other steps to get the image in the right place?

**Comment by doanac** [comment]
@gcg-foundries - i think its just your browser cache. I pulled up the link and it was what i expected.

File: source/tutorial/basic-system.rst [hunk 1]
```diff
@@ -40,6 +45,15 @@ Prepare the System
...
 ------------------
 
+We use the Nordic nRF52840 Dongle device to provide 802.15.4 communications
+with OpenThread. Follow the `directions`_ on the Nordic web site to program 
+the Dongle with the OpenThread NCP firmware using the nRFConnect programmer. 
+A ready to install `binary file`_ is available from Foundries.io. You may 
+also build this yourself from the github.com OpenThread project. Note that 
+once installed this dongle may be used on any gateway using the LmP that 
+supports the reference software including x86 and Arm-based platforms. 
+Install the programmed Dongle into any of the Raspberry Pi USB ports.
```
Comment (mike-scott) [inline source/tutorial/basic-system.rst]: This section needs some work:
- directions on the Nordic web site to program the Dongle aren't exactly straight forward.  I always use the "nRF Connect" application (for Linux or Windows) with the "Programmer" add-in application.
- I'm debating creating a page in our own docs to outline how to build the OpenThread NCP firmware and flash it to the dongle.
- Also, the binary file link is to our private drive.  IMHO, if we want to host a file, then we should build it as a part of each release artifacts.

File: source/tutorial/basic-system.rst [hunk 2]
```diff
@@ -80,24 +99,30 @@ Then, build and flash the basic LWM2M system's application.
    communication security. You can enable DTLS-based security on
    nRF52840 devices using instructions in the next document in this
    tutorial.
+   
+By default the system uses 802.15.4 OpenThread nesh for
```
Comment (mike-scott) [inline source/tutorial/basic-system.rst]: s/nesh/mesh/

File: source/tutorial/basic-system.rst [hunk 3]
```diff
@@ -80,24 +99,30 @@ Then, build and flash the basic LWM2M system's application.
...
-   .. tab-container:: nrf52840_pca10056
-      :title: nRF52840 DK
+         
+   .. tab-container:: nrf52840_pca10056_BLE
+      :title: nRF52840-DK Bluetooth
 
       .. code-block:: console
 
-         west build -s zmp-samples/dm-lwm2m -d build-dm-lwm2m -b nrf52840_pca10056
+         west build -s zmp-samples/dm-lwm2m -d build-dm-lwm2m -b nrf52840_pca10056 -- -DOVERLAY_CONFIG=overlay-ble.conf
```
Comment (mike-scott) [inline source/tutorial/basic-system.rst]: Change to:
`west build -s zmp-samples/dm-lwm2m -d build-dm-lwm2m -b nrf52840_pca10056 -- -DCONFIG_FOTA_NET_BLE6LOWPAN=y`

File: source/tutorial/dtls-system.rst [hunk 4]
```diff
@@ -80,5 +81,5 @@ https://mgmt.foundries.io/leshan/#/security
 Use the System
 --------------
 
-You should now be able to use the system with DTLS enabled in
+You should now be able to use the system with DTLS enabled in the
```
Comment (mike-scott) [inline source/tutorial/dtls-system.rst]: This page also has example commands for building the DTLS binary:
`west build --board reel_board -s zmp-samples/dm-lwm2m -d build-dm-lwm2m-dtls -- -DOVERLAY_CONFIG=overlay-dtls.conf`
can we change this to the DK board:
`west build -s zmp-samples/dm-lwm2m -d build-dm-lwm2m-dtls -b nrf52840_pca10056 -- -DOVERLAY_CONFIG=overlay-dtls.conf`

File: source/tutorial/other-zephyr-boards.rst [hunk 5]
```diff
@@ -6,13 +6,20 @@ Using Other Zephyr Boards
 .. warning:: Content in this section is provided on a best-effort basis.
```
Comment (mike-scott) [inline source/tutorial/other-zephyr-boards.rst]: There's a section on this page for K64F:
To use DHCP, with gateway IP address A.B.C.D, create the file with the following contents.

DHCP is now enabled by default and the only thing the user needs to set is:
CONFIG_NET_APP_PEER_IPV4_ADDR="A.B.C.D"

Comment (mike-scott) [reply source/tutorial/other-zephyr-boards.rst]: And the paragraph:
To use a static IP address X.Y.Z.W for the FRDM-K64F instead, use this.

Should include a line for:
CONFIG_NET_DHCPV4=n

**Comment by mike-scott** [comment]
I think we can probably iron out the details for all but the Dongle flashing instructions with this PR.  And then fixup the rest in a new PR.

**Comment by mike-scott** [comment]
Closing in favor of a new PR.

**Pull request closed without merging**